malaria-ios/ViewControllers/AchievementsViewController.swift

```diff
@@ -1,10 +1,98 @@
 import UIKit
 
 class AchievementsViewController: UIViewController {
+  typealias AchievementObject = (sectionName: String, achievements: [Achievement])
+  
+  @IBOutlet weak var tableView: UITableView!
+  
+  // The array that will hold each section's name and its achievements.
+  var allAchievements: [AchievementObject] = []
   
   override func viewDidLoad() {
     super.viewDidLoad()
     
     view.backgroundColor = UIColor(patternImage: UIImage(named: "background")!)
+    
+    tableView.estimatedRowHeight = 75.0
+    tableView.rowHeight = UITableViewAutomaticDimension
+  }
+  
+  override func viewWillAppear(animated: Bool) {
+    super.viewWillAppear(animated)
+    updateModel()
+  }
+  
+  func updateModel() {
+    allAchievements = []
+    
+    let achievementManager = AchievementManager.sharedInstance
+    
+    let gamesAchievements =
```

Comment:
Group together:
```swift
 let manager1 = (...)
appendToModel(...)

let manager2 = (...)
appendToModel(...)
```

Instead of:
```swift
let manager1 = (...)
let manager2 = (...)

appendToModel(...)
appendToModel(...)
```